app/handlers/profiles/settings/profbackground.go

```diff
@@ -74,7 +87,29 @@ func ProfileBackgroundSubmitHandler(c *gin.Context) {
 			slog.ErrorContext(c, err.Error())
 			return
 		}
-		saveProfileBackground(ctx, 1, fmt.Sprintf("%d.jpg?%d", ctx.User.ID, time.Now().Unix()))
+		// seek file to beginning
+		f.Seek(0, io.SeekStart)
+
+		sess := session.Must(session.NewSession(&aws.Config{
+			Region:   aws.String(settings.AWS_REGION),
+			Endpoint: aws.String(settings.AWS_ENDPOINT_URL),
+		}))
+		uploader := s3manager.NewUploader(sess)
+		_, err = uploader.Upload(&s3manager.UploadInput{
+			Bucket:      aws.String(settings.AWS_BUCKET_NAME),
+			Key:         aws.String(fmt.Sprintf("profile-backgrounds/%d.jpg", ctx.User.ID)),
```

Comment:
i think u will need to create the directory in wasabi ui

Reply:
i don't think that's how it works?  but not confident.. can u do it, i'm on mobile

Reply:
yup will do it just in case

also we'll need to backfill here

Reply:
yep we can do that at some point

Reply:
done